src/tiled/changemapobject.cpp

```diff
@@ -203,3 +203,63 @@ void DetachObjects::undo()
 
     emit mMapDocument->mapObjectModel()->objectsChanged(mMapObjects);
 }
+
+ResetInstances::ResetInstances(MapDocument *mapDocument,
+                               QList<MapObject *> mapObjects,
+                               QUndoCommand *parent)
+    : QUndoCommand(QCoreApplication::translate("Undo Commands",
+                                               "Reset %n Instnaces",
+                                               nullptr, mapObjects.size()), parent)
+    , mMapDocument(mapDocument)
+    , mMapObjects(mapObjects)
+{
+    for (const MapObject *object : mMapObjects) {
+        mProperties.append(object->properties());
+        mChangedProperties.append(object->changedProperties());
```

Comment:
`mChangedProperties` isn't actually used.

Reply:
Okay, even `mProperties` isn't used, I didn't think I will clone the objects when I created them.